Examples/Algorithms/TrackFittingChi2/src/TrackFittingChi2Options.cpp

```diff
@@ -0,0 +1,19 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ActsExamples/TrackFittingChi2/TrackFittingChi2Options.hpp"
+
+#include "ActsExamples/Utilities/Options.hpp"
+
+using namespace boost::program_options;
```

Comment:
I see that the other `addOptions()` in the code base use `ActsExamples::Options::Description` instead of `boost::program_options::options_description`. But as we would also need to change it in the KF, this should go in another PR, I think…

Reply:
I see, thank you. Since we have similar `using namespace` cases in numerous other places, I think we could keep it as it is for now and postpone the decision to a future PR.